deployment/templates/es.yaml

```diff
@@ -42,12 +42,7 @@ objects:
             kind: ImageStreamTag
             name: logging-elasticsearch:latest
       strategy:
-        resources: {}
-        rollingParams:
-          intervalSeconds: 1
-          timeoutSeconds: 600
-          updatePeriodSeconds: 1
-        type: Rolling
+        type: Recreate
```

Comment:
There aren't any needed for recreate (it's pretty simple).
https://docs.openshift.com/enterprise/3.0/dev_guide/deployments.html#recreate-strategy

On Fri, Oct 30, 2015 at 2:11 PM, Eric Wolinetz <notifications@github.com>
wrote:

> In deployment/templates/es.yaml
> <https://github.com/openshift/origin-aggregated-logging/pull/15#discussion_r43534427>
> :
>
> > @@ -42,12 +42,7 @@ objects:
> >              kind: ImageStreamTag
> >              name: logging-elasticsearch:latest
> >        strategy:
> > -        resources: {}
> > -        rollingParams:
> > -          intervalSeconds: 1
> > -          timeoutSeconds: 600
> > -          updatePeriodSeconds: 1
> > -        type: Rolling
> > +        type: Recreate
>
> I forget, does this also need recreate params as well?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin-aggregated-logging/pull/15/files#r43534427>
> .
>